.github/workflows/build-solutions.yml

```diff
@@ -41,3 +42,9 @@ jobs:
         run: dotnet build EthereumTests.sln -c ${{ env.BUILD_CONFIG }} --no-restore
       - name: Build Benchmarks.sln
         run: dotnet build Benchmarks.sln -c ${{ env.BUILD_CONFIG }} --no-restore
+      - name: Build Nethermind.sln
+        run: dotnet build Nethermind.sln -c ${{ env.SECOND_BUILD_CONFIG }} --no-restore
+      - name: Build EthereumTests.sln
+        run: dotnet build EthereumTests.sln -c ${{ env.SECOND_BUILD_CONFIG }} --no-restore
+      - name: Build Benchmarks.sln
+        run: dotnet build Benchmarks.sln -c ${{ env.SECOND_BUILD_CONFIG }} --no-restore
```

Comment:
Shall we put it as a separate job, so it will run in parallel?

Reply:
> The workflow fix will likely require changes in the PR required actions list.

That's what I mean.